### ISSUE TITLE
some detail about finetuning

### ISSUE THREAD

**Issue by King-king424** [slei109/PATNet#42]
![image](https://github.com/slei109/PATNet/assets/39642013/21d03f96-0a2f-4f0b-9158-7a0ac6642d0c)
I wonder how M_hat got it. The decoder outputs a feature map of size [b,2, h, w], but the size of the M_hat in the paper seems to be [b, 1, h, w].

**Comment by King-king424** [comment]
And could you provide the complete test process code, including test-time finetuning? It may be that I have some issues with the code implementation and test-time finetuning is not working. It would be of great help to me if you could provide the full test process code.

**Comment by slei109** [comment]
We calculated the M_hat via the output of the softmax layer.
